app/scripts/modules/amazon/serverGroup/configure/serverGroupCommandBuilder.spec.js

```diff
@@ -26,6 +27,7 @@ describe('awsServerGroupCommandBuilder', function() {
 
     it('initializes to default values, setting usePreferredZone flag to true', function () {
       var command = null;
+      this.settings.providers.aws.defaults.iamRole = '{{application}}IAMRole';
```

Comment:
I need to think about this some more tomorrow.  There should be a cleaner way of doing parameterized strings.